.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Build on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        uses: actions/checkout@v2
+        name: Set up QEMU
```

Comment:
Bit of a nit with this formatting, can we combine the first two lines of each of these steps?